benchmark/v5.1-performance-benchmarking-with-tpch.md

```diff
@@ -112,4 +112,4 @@ Apache Spark 测试使用 Apache Parquet 作为存储格式，数据存储在 HD
 
 ![TPC-H](/media/tidb-v5-tpch-100-vs-gp-spark.png)
```

Comment:
@ilovesoup Would you please share the Google sheets with this graphic so we can update v5.0 in the graphic to v5.1?

Reply:
@qiancai A simple image processing can also do the trick. PTAL https://github.com/pingcap/docs-cn/blob/eeda5575e0a710443456c1ea6d4b2f4bdd352d27/media/tidb-v5-tpch-100-vs-gp-spark.png